settings.gradle

```diff
@@ -1,14 +1,28 @@
+pluginManagement {
+    repositories {
+        gradlePluginPortal()
+        google()
+        mavenCentral()
+    }
+}
+dependencyResolutionManagement {
+    repositoriesMode.set(RepositoriesMode.FAIL_ON_PROJECT_REPOS)
+    repositories {
+        google()
+        mavenCentral()
+        maven { url "https://jitpack.io" }
+    }
+}
+rootProject.name = "nRF Blinky"
 include ':app'
+include ':scanner'
+include ':blinky:spec'
+include ':blinky:ui'
+include ':blinky:ble'
 
-// To include Android Scanner Compat Library project as a module, clone it to
-// Android-Scanner-Compat-Library folder.
-
-if (file('../Android-Scanner-Compat-Library').exists()) {
-    includeBuild('../Android-Scanner-Compat-Library')
+if (file('../Android-Common-Libraries').exists()) {
```

Comment:
The project doesn't build when I comment this and I cannot build it because I have locally different head for Common-Libraries.